colossalai/kernel/triton/context_attn_unpad.py

```diff
@@ -220,7 +217,7 @@ def context_attention_unpadded(
     assert block_size in {16, 32, 64, 128}
     BLOCK_M = BLOCK_N = block_size
 
-    grid = (num_seqs, num_heads, triton.cdiv(max_seq_len, BLOCK_M))
+    grid = (triton.next_power_of_2(num_seqs), num_heads, triton.cdiv(max_seq_len, BLOCK_M))
```

Comment:
Why do this?

Reply:
Prevent compiling every time we meet a new batch size - Decreasing the number of compile times. Triton will cache based on grid size.